### PR TITLE
Dashboard discover movies

### LINKED ISSUE #12: Dashboard: Discover Movies
As a authenticated user, 
When I click on the "Discover Movies" button,
I am redirected to a discover page.

### PR DESCRIPTION
**Pull request by Eternal-Flame085**
- Created Dashboard button "Discover Movies"
- Created spec for dashboard
- Created spec for Discover movies view
- Created the discover movies view page

Closes #12

### CODE REVIEW

File: spec/features/dashboard/discover_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  before :each do
+    User.create(email: 'test@gmail.com', password: 'test', first_name: 'Alex', last_name: 'Rivero')
+
+    visit  '/'
+    fill_in :email,	with: "test@gmail.com"
+    fill_in :password,	with: "test"
+    click_button 'Sign In'
+  end
+
+  it 'when I visit /discover' do
```
Comment (stillsheryl) [inline spec/features/dashboard/discover_spec.rb]: Should we expand on what we expect here?

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
+  helper_method :current_user
```
Comment (stillsheryl) [inline app/controllers/application_controller.rb:4]: Nice catch adding this.

**Review by Stearnzy — APPROVED** [review]
Looks great

**Review by stillsheryl — APPROVED** [review]
Looks great!

**Pull request merged**